### PR TITLE
Correct docstrings and a typo in comments

### PR DESCRIPTION
**Pull request by lewis-yeung**
## What does this PR do?

Correct docstrings of the following methods:

- `TrainingArguments.set_save`
- `TrainingArguments.set_logging`

## Who can review?

@stevhliu

### CODE REVIEW

File: src/transformers/training_args.py [hunk 1]
```diff
@@ -2276,7 +2276,7 @@ def set_save(
         on_each_node: bool = False,
     ):
         """
-        A method that regroups all arguments linked to the evaluation.
+        A method that regroups all arguments linked to the checkpoint saving.
```
Comment (stevhliu) [inline src/transformers/training_args.py:2279]:
```suggestion
        A method that regroups all arguments linked to checkpoint saving.
```

File: src/transformers/training_args.py [hunk 2]
```diff
@@ -2329,7 +2329,7 @@ def set_logging(
         replica_level: str = "passive",
     ):
         """
-        A method that regroups all arguments linked to the evaluation.
+        A method that regroups all arguments linked to the logging.
```
Comment (stevhliu) [inline src/transformers/training_args.py:2332]:
```suggestion
        A method that regroups all arguments linked to logging.
```

**Review by stevhliu — APPROVED** [review]
Good catch, thanks for the fixes!

File: src/transformers/training_args.py
Comment (lewis-yeung) [inline src/transformers/training_args.py:1]: @stevhliu Should we change this (remove `the` from `the evaluation`) as well?

https://github.com/huggingface/transformers/blob/6cbc1369a330860c128a1ba365f246751382c9e5/src/transformers/training_args.py#L2181

Comment (stevhliu) [reply src/transformers/training_args.py:1]: Sure, that'll keep it nice and consistent :)

Comment (lewis-yeung) [reply src/transformers/training_args.py:1]: Updated. Also fixed a typo (`sur` -> `sure`) in `trainer.py`:

https://github.com/huggingface/transformers/blob/a2f55a65cd0eb3bde0db4d5102a824ec96c7d7e9/src/transformers/trainer.py#L1904

**Pull request merged**